kano_updater/apt_progress_wrapper.py

```diff
@@ -62,18 +62,16 @@ def __init__(self, updater_progress, ops=[]):
         self._updater_progress = updater_progress
         self._phase_name = updater_progress.get_current_phase().name
 
-        ops = [self._get_op_key(op) for op in ops]
+        ops = [self._get_op_key(op[0]) for op in ops]
 
-        phases = [Phase(op, op) for op in ops]
+        phases = [Phase(op[0], op[1]) for op in ops]
```

Comment:
I think that this line isn't right because `ops` has now been overwritten so it is no longer a list of tuples. Either change the above to

```python
 ops = [(self._get_op_key(op[0]), op[1]) for op in ops]
```

or rename the variable and use some sort of `zip()` operation.